projects/storefrontstyles/scss/components/misc/chat-messaging/_avatar.scss

```diff
@@ -0,0 +1,25 @@
+%cx-avatar {
```

Comment:
the styles looks fine to me

I don't see a problem, but since it's for a shared component, where it's going to be used across different featuress (potentially), then send this PR to Miguel/Mikel if they like what they see

Reply:
The styling is missing the css variables for font and colors. We use variables because spartacus has the ability to switch themes. If you got0 your console in the browser and add the class `santorini` to the `cx-storefront` tag you'll see the theme change. I'll add the comments in the scss with the different vars.